barcode-network/roadsidereporting
# Fix: resolve UI issues

Resolve error prohibiting the deployment of the master branch due to a merged merge conflict 

## Description
This PR makes the following changes:

1. Resolves a markup error in `src/components/Map.js` caused by an accidentally merged merge conflict
2. Resolve a linting error in `src/components/Header.js` by replacing `class` with `className`

## Related Issue / Asana Task
N/A

## Motivation and Context
The app was not compiling or starting

## How Has This Been Tested?
These changes can be tested as follows

1. Switch to the `fix/resolve-ui-issues` branch.
2. Run `yarn start` in the command line
3. The app should compile and auto-launch in-browser

## Screenshots (if appropriate):
N/A
